### PR TITLE
Anandbhagwatt32gmail.com pune

### PR DESCRIPTION
**Pull request by anandbhagwat**
testchange

### CODE REVIEW

File: _pins/githubstudnet.json [hunk 1]
```diff
@@ -0,0 +1,2 @@
+---
```
Comment (crichID) [inline _pins/githubstudnet.json]: @anandbhagwat if you remove this `_pins/githubstudent.json` file from your pull request, the build should pass! Please let us know if you need any assistance.

Comment (anandbhagwat) [inline _pins/githubstudnet.json]: I am very newbie on the github accendentuily done the requestsv didnt understood anything,sorry for inconvience

Comment (crichID) [inline _pins/githubstudnet.json]: @anandbhagwat it is not an inconvenience - we all have to start learning sometime! Removing the file can be tricky so I will do it really quick and then you can merge the pull request.

**Comment by crichID** [comment]
@anandbhagwat I went ahead and fixed the syntax issues in https://github.com/githubschool/open-enrollment-classes-introduction-to-github/pull/4518/commits/fb099e5761071ee0a90abc185de6ed32095d678e. Please take a quick look at the changes. Once the checks have finished, please feel free to click the big green **Merge pull request** button. If you haven't merged it by March 17, 2017, I will go ahead and do it just to keep the repository clean!

**Pull request merged**